src/scss/_filters.scss

```diff
@@ -189,3 +189,8 @@ button.disclosure-btn[aria-expanded=true]::after  {
   content: "\e5ce";
 }
 
+// TODO: obnoxious, isn't this?
+::highlight(search-results) {
+  background-color: #f06;
+  color: white;
+}
```

Comment:
for doc site, I think it is:
background-color: var(--color-maize-300);
color: var(--color-neutral-400);
font-weight: var(--semibold);

Using the bold stays away from utilizing color alone and it meets contrast.